spec-0004/index.md

```diff
@@ -91,26 +91,36 @@ Complete examples of how projects implement this in their CI setup are linked in
 #### Process for Adding New Projects
 
 After someone creates an issue on https://github.com/scientific-python/upload-nightly-action
-requesting access to upload wheels a human has to respond to that request.
+requesting access to upload wheels a human/admin has to respond to that request.
+
+Admins are people from the community who are ideally not part of the same organizations nor
+project. This is to prevent malicious activities from a given group of actors and ensure a
+diverse and healthy community. Adding new people to the list requires of admins requires at
+least an issue to be openned.
 
 We want to be open to projects uploading wheels but at the same time need to perform some
 amount of due dilligence before giving people access. This is because once a user is given
-access they could upload wheels for any project. We assume that people are not malicious
-and we can see from the logs who misbehaved after the fact.
+access they benefit the wide exposure of Scientific Python and could take this to their
+advantage to publish malicious packages.
```

Comment:
```suggestion
access their work will be broadcasted through the broad exposure of Scientific Python. This
could be abused to publish malicious packages. 
```

Reply:
I think this was the essence of what you were trying to say. I rephrased for clarity but am happy to hear alternative suggestions.

Reply:
Yes thanks!